babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,15 +788,71 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList paramNames = new SqlNodeList(getPos());
+    SqlNodeList paramValues = new SqlNodeList(getPos());
     Span s;
 }
 {
     macro = CompoundIdentifier() { s = span(); }
+    [
+        SqlExecMacroArgument(paramNames, paramValues)
+    ]
     {
-        return new SqlExecMacro(s.end(this), macro);
+        return new SqlExecMacro(s.end(this), macro, paramNames, paramValues);
     }
 }
 
+void SqlExecMacroArgument(SqlNodeList paramNames, SqlNodeList paramValues) :
+{
+    SqlNode name;
+    SqlNode value;
+}
+{
+        <LPAREN>
+        (
+            value = Literal()
+            {
+                paramValues.add(value);
+            }
+
+        )
```

Comment:
Also no need for these parentheses (812 & 818), and extra space on 817.

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/108/commits/41eff94023a11dbee3d010b4db619eae40217431